### ISSUE TITLE
[priest] Mindvision wird durch Stealh gebrochen

### ISSUE THREAD

**Issue by stva23** [ClassicWoW/Nefarian_1.12.1_Bugtracker#2483]
**Welches Verhalten wird beobachtet?**
Wird ein Schurke oder eine Katze mit Mindvision belegt, wird der Zauber unterbrochen, wenn das Ziel in die Verstohlenheit wechselt.

**Wie sollte es sich verhalten?**
Mindvision sollte nicht gebrochen werden.

**Schritte zur Reproduzierung**
Mindvision auf einen gegnerischen Schurken / eine Katze casten, warten bis der Gegenspieler Stealth benutzt.

**Zusätzliche Informationen (Screenshots, Videos, Klasse, Rasse, Level, etc.)**
z.B https://www.youtube.com/watch?v=hW46h_q68lE gibt aber noch mehr auf YT

**Alle Kreaturen, Items, Objekte, Quests, Zauber etc. müssen aus unserer Datenbank verlinkt sein.**
[MindVision](https://datenbank.classic-wow.org/?spell=10909)

**PS:**
Vielleicht geht es ja in einem mit #2395

**Comment by Dotox** [comment]
03.01.2007 ist 2.0.1

**Comment by stva23** [comment]
Hallo, 
BC kam erst am 15, bzw. 16 Jänner 2007.

Siehe [hier](http://wowwiki.wikia.com/wiki/Expansion) oder [hier](https://de.wikipedia.org/wiki/World_of_Warcraft#Verkaufszahlen).

**Comment by Dotox** [comment]
Richtig, der Pre-Patch kam trotzdem früher und daher ist 2.0.1 ist mehr Classic.
